C/Cpp_include/c4Collection.hh

```diff
@@ -64,6 +65,13 @@ struct C4Collection
 
     static C4Document* documentContainingValue(FLValue) noexcept;
 
+    /**
+ * Default arguments on virtual methods are strongly argued against by the C++ standards, Google goes so far as to
```

Comment:
The indent seems off.

Reply:
I think Clang-Format wanted to align the comment with the code, this comment is removed in latest commit now anyway.